app/javascripts/main/store.js

```diff
@@ -43,7 +44,8 @@ class Store {
 
 function parseDataFile(filePath, defaults) {
   try {
-    return JSON.parse(fs.readFileSync(filePath));
+    const userData = JSON.parse(fs.readFileSync(filePath));
+    return Object.assign(defaults, userData);
```

Comment:
This is needed so `isMenuBarVisible` doesn't get stored as `undefined` when people upgrade from a previous version.